lib/snapshot/BUILD.gn

```diff
@@ -197,6 +197,29 @@ bin_to_linkable("platform_strong_dill_linkable") {
   executable = false
 }
 
+action("create_arm_gen_snapshot") {
```

Comment:
@jmagman @cbracken This rule must declare a dependency on some rule that causes gen_snapshot to end up at the expected location. Otherwise GN/Ninja might try to run this rule before gen_snapshot is there.

Reply:
I had to reopen https://github.com/flutter/flutter/issues/38933 since it didn't work the next time I ran it.  That would explain it...

Reply:
I am poking around something related, and will make a concrete suggestion shortly.